### PR TITLE
Tag sidebar#22

### PR DESCRIPTION
**Pull request by tgross-usgs**
changed sidebar panel functionality, added listener in StudyService, set up Jobsons modal

### CODE REVIEW

File: src/app/modules/core/components/sidebar/sidebar.component.ts [hunk 1]
```diff
@@ -167,21 +134,36 @@ export class SidebarComponent {
     try {               
         switch (pType) {
             case ProcedureType.MAPLAYERS:
+              if(this.SelectedProcedureType === 0) {
```
Comment (tim7en) [inline src/app/modules/core/components/sidebar/sidebar.component.ts:154]: keep accordion panel open  at all times to guide the user

File: src/app/modules/core/components/sidebar/sidebar.component.ts [hunk 2]
```diff
@@ -167,21 +134,36 @@ export class SidebarComponent {
...
+                  this.SetProcedureType(4);
+                  return false;
                 }
                 return true;
             case ProcedureType.REPORT:
-                if(!this.SelectScenario) {
-                  throw new Error("Can not proceed until study area options are selected.")
-                } else if (!this.ReportIsActive) {
-                   throw new Error("Can not proceed until Scenario options are selected.")
+                if(!this.StudyService.selectedStudy || this._step !== 4) return;
```
Comment (tim7en) [inline src/app/modules/core/components/sidebar/sidebar.component.ts]: change this._step method from observable to property type

**Review by tim7en — CHANGES_REQUESTED** [review]
see comments, sidebar
please include study and studyservice in your next push

File: src/app/modules/core/components/sidebar/sidebar.component.ts (hunk 1 above)
Comment (tgross-usgs) [reply src/app/modules/core/components/sidebar/sidebar.component.ts:154]: Created a previousProcedureType to always revert to the previously opened panel when user closes a panel - except in the case that map layers was the previously opened panel in which case the active panel remains open.

**Pull request closed without merging**